{{cookiecutter.repostory_name}}/letsencrypt_setup.sh

```diff
@@ -6,16 +6,17 @@ ABSPATH="$(realpath "$RELPATH")"
 cd "$ABSPATH"
 
 source ./.env
+mkdir -p $ABSPATH/letsencrypt/etc/dhparams
```

Comment:
```suggestion
mkdir -p "$ABSPATH/letsencrypt/etc/dhparams"
```
or it'll explode because of a space one day